modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/I18n/Language.json

```diff
@@ -434,6 +437,7 @@
 	"tcat-portal-non production": "TCAT Portal Non-Production",
 	"tcat-portal-production": "TCAT Portal Production",
 	"team-member": "Team Member",
+	"team-member-x": "Team Member:",
```

Comment:
can we move the colon out of the language key and into the html instead, bchan usually prefers it that way

Reply:
Fixed, thank you!